deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```
@@ -37,7 +37,8 @@ metadata:
                       "pvcStorageRequest": "20Gi"
                     },
                     "strategy": "persistent"
-                  }
+                  },
+                  "version": "7.10.2"
```

```suggestion
                  "version": "7.16.1"
```

To match https://github.com/infrawatch/service-telemetry-operator/pull/303/files#diff-d511e7d4a1c44b35b99cee004163a2e03c2f63c22077432de49c8f668e7a381fR42